tests/unit/amo/components/TestUserProfileEdit.js

```diff
@@ -1254,7 +1254,7 @@ describe(__filename, () => {
     expect(modal).toHaveLength(1);
     expect(modal).toHaveProp(
       'header',
-      'IMPORTANT: Deleting your account is irreversible.',
+      'IMPORTANT: Deleting your Firefox Add-ons profile is irreversible.',
```

Comment:
Similar to my comment above, this is essentially static text, except for the fact that it conditionally says either "your Firefox Add-ons profile" or "this Firefox Add-ons profile", so it might ease the maintenance burden if we just check that the `header` contains the part of the string we expect to change.

Although maybe that will make the test more confusing, so I suppose either way is suboptimal. 🤷‍♂️